src/mscorlib/src/System/Array.cs

```diff
@@ -976,7 +976,7 @@ public Object Clone()
             if (array==null)
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.array);
             Contract.EndContractBlock();
-            return BinarySearch<T>(array, 0, array.Length, value, null);
+            return BinarySearch<T>(array, 0, array.Length, value, (IComparer<T>)null);
```

Comment:
@karelz I think the up-for-grabs tag on api-needs-review issues is confusing or at least needs to be documented